engine/src/test/java/io/camunda/zeebe/engine/processing/incident/ServiceTaskIncidentTest.java

```diff
@@ -286,4 +286,49 @@ public void shouldResolveIncidentAfterJobRetriesExpressionEvaluationFailed() {
 
     assertThat(incidentResolvedEvent.getKey()).isEqualTo(incidentRecord.getKey());
   }
+
+  @Test
+  public void shouldResolveIncidentAndCreateNewIncidentWhenContinuationFails() {
+    // given a deployed process with a service task with an output expression
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            Bpmn.createExecutableProcess("process-incident-on-leaving-task")
+                .startEvent()
+                .serviceTask(
+                    "task",
+                    b -> b.zeebeJobType("task").zeebeInputExpression("unknown_var", "output"))
```

Comment:
I'm a bit confused because this is an input expression and the comments talk about an output expression

Reply:
Good point. I started out with an output expression, but changed it later. I'll make sure to update it

Reply:
This is still open

Reply:
I undated only part of it, it seems. Should be fixed now